src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -274,19 +320,75 @@ object VEP {
               printElement,
               _ => ())
 
-            val kt = if (csq)
-              jt.filter(s => !s.isEmpty && s(0) != '#')
-                .map { s =>
-                  val x = csq_regex.findFirstIn(s).get
-                  (variantFromInput(s), x.substring(4))
+            val original_variant = block.map({
```

Comment:
Scala variables should be camelCase.  `originalVariant`.  Actually, the name should indicate what it maps to/from.  Maybe `nonStarToOriginalVariant`?  Parens are unnecessary for the call to `map`.